src/diffusers/loaders.py

```diff
@@ -1236,9 +1310,12 @@ def _convert_kohya_lora_to_diffusers(cls, state_dict):
         unet_state_dict = {}
         te_state_dict = {}
         network_alpha = None
+        unloaded_keys = []
 
         for key, value in state_dict.items():
-            if "lora_down" in key:
+            if "hada" in key or "skip" in key:
```

Comment:
out of curiosity what is `"hada"` ?

Reply:
Its a key from a new format that we don't support yet, but more context can be found here: https://github.com/huggingface/diffusers/issues/4133#issuecomment-1641209337